File: scala/sbt/sbt_0_11/sample_servlet/src/main/scala/SampleServlet.scala
import javax.servlet.http._

class SampleServlet extends HttpServlet {

	val html = 
<html>
<head>
	<title>Scala Servlet Sample</title>
</head>
<body>
	<h1>Sample</h1>
</body>
</html>

	override def doGet(req: HttpServletRequest, res: HttpServletResponse) = {
		res.getWriter().print(html);
	}
}

File: blog/20140203/gradle-jetty/src/main/java/fits/sample/SampleServlet.java
package fits.sample;

import java.io.IOException;

import javax.servlet.ServletException;
import javax.servlet.annotation.WebServlet;
import javax.servlet.http.HttpServlet;
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;

@WebServlet(urlPatterns = {"/sample"})
public class SampleServlet extends HttpServlet {
	@Override
	protected void doGet(HttpServletRequest req, HttpServletResponse res) throws ServletException, IOException {

		res.getWriter().print("sample");
	}
}